ui/src/common/components/CitationsPapersIcon.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoginOutlined } from '@ant-design/icons';
+
+import IconText from './IconText';
+import FormattedNumber from './FormattedNumber';
+import pluralizeUnlessSingle from '../utils';
+
+const CitationsPapersIcon = ({ itemCount, displayedName }) => (
```

Comment:
name doesn't make sense for `experiments papers` right? or `conference papers` which is not using at the moment but it should.

On the other hand better approach would be to single component for all.
(name could be different)
```
<IncomingLiteratureReferencesLinkAction
  count={1234}
  referenceType="contribution or paper or citation etc"
  linkQuery={...the query that will be appended to the literature link...}
/>
```